api/views.py

```diff
@@ -740,10 +742,13 @@ def post(self, request, format=None):
                 user.profile.is_referred = True
                 user.profile.referred_by = queryset[0]
 
+            user.profile.wants_stealth = False
```

Comment:
Maybe good idea to leave here. But I believe it is not needed since `user.profile.wants_stealth` is already `False` by default